examples/amp-consent/diy-consent.html

```diff
@@ -261,6 +262,7 @@ <h1>Thank you!</h1>
     type: 'consent-response',
     action: 'accept',
     info: 'accept-str',
+    consentStringType: 'tcf-v2',
```

Comment:
Given there will be other metadata besides consentStringType, I want to explore more ways on how to group them together. 
Let's discuss offline tomorrow.